src/appleseed/foundation/utility/statistics.h

```diff
@@ -194,7 +196,14 @@ class Statistics
     void insert(
         const std::string&              name,
         const Population<T>&            value,
-        const std::string&              unit);
+        const std::streamsize           precision);
```

Comment:
Why no default value here?

Reply:
To fallback on the default insert method which doesn't know the type